awscrt/io.py

```diff
@@ -281,3 +283,30 @@ def set_alpn_list(self, alpn_list):
 
     def set_server_name(self, server_name):
         _awscrt.tls_connection_options_set_server_name(self, server_name)
+
+
+class InputStream(NativeResource):
```

Comment:
consideration, support for with .... autoclose, whatever it's called.

Reply:
This class take a python `IOBase` instance in its constructor. It would be up to whoever opened *that* instance to close that instance.